Terminal-iOS/Study/CreateStudy/LocationView/SelectLocation/SelectLocationRemoteDataManager.swift

```diff
@@ -8,10 +8,62 @@
 
 import UIKit
 import NMapsMap
+import Alamofire
+import SwiftyJSON
 
-class SelectLocationRemoteDataManager: SelectLocationRemoteDataManagerProtocols {
-    func getAddressWithLatLng(latLng: NMGLatLng?) -> (NMGLatLng, String) {
-        print("getAddressWithLatLng")
-        return (NMGLatLng(), "")
+class SelectLocationRemoteDataManager: SelectLocationRemoteDataManagerProtocol {
+    func getAddressInfo(lat: Double, lng: Double, completion: @escaping (Bool, searchLocationResult?) -> ()) {
+        //api ㅆ고
+        let headers: HTTPHeaders = [ "Authorization": "KakaoAK 6cd40b04c090b1a033634e5051aab78c" ]
+        let parameters: Parameters = [
+            "x" : lng,
+            "y" : lat
+        ]
+        
+        AF.request("https://dapi.kakao.com/v2/local/geo/coord2address.json",
+                   method: .get,
+                   parameters: parameters, headers: headers).responseJSON(completionHandler: { [self] response in
```

Comment:
```suggestion
        AF.request("https://dapi.kakao.com/v2/local/geo/coord2address.json",
                   method: .get,
                   parameters: parameters,
                   headers: headers).responseJSON(completionHandler: { [self] response in
```

스타일 가이드에 대해서 다시 이야기 나눠볼 필요가 있을듯 합니다. 

매개변수가 길어짐에 따라서 한줄이상 차지해 개행한다면 파라미터 별로 나누어 표현하는게 좋을거 같은데 다음에 이야기 해보는걸로 하죵

Reply:
좋슴돠~~~~